draft-ietf-tls-external-psk-importer.md

```diff
@@ -87,6 +87,36 @@ a tuple of (Base Key, External Identity, KDF). The associated KDF (and hash func
 - External Identity: The identity of an EPSK.
 - Imported Identity: The identity of a PSK as sent on the wire.
 
+# Binder Key
+To prevent PSK Importers from being confused with standard OOB PSKs we change the label used in the computation of the PSK binder key.
+In TLS 1.3 the PSK binder key computation is defined as follows:
+
+                 0
+                 |
+                 v
+       PSK ->  HKDF-Extract = Early Secret
+                 |
+                 +-----> Derive-Secret(., "ext binder" | "res binder", "")
+                 |                     = binder_key
+
+We replace the string "ext binder" with "imp binder".
+This means the binder key is now computed as follows:
+
+                 0
+                 |
+                 v
+       PSK ->  HKDF-Extract = Early Secret
+                 |
+                 +-----> Derive-Secret(., "ext binder"
+                                        | "res binder"
+                                        | "imp binder", "")
```

Comment:
Super nitpicky nitpick: there are some `|`s missing in the ASCII diagram. The left vertical line got interrupted in the line wrapping.